WikiLookup.json

```diff
@@ -575,5 +575,46 @@
 			"Star Citizen",
 			"Squadron 42"
 		]
+	},
+		{
+		"name": "Unofficial Elder Scrolls Pages",
+		"homepage": "https://en.uesp.net	",
+		"lang": "en",
+		"api": "https://en.uesp.net/w/api.php",
+		"companies": [
+		    "Bethesda Softwarks",
+			"Zenimax Online Studios"
+		],
+		"games": [
+		    "Arena",
+			"Daggerfall",
+			"Redguard",
+			"Battlespire",
+			"Morrowind",
+			"Oblivion",
+			"Skyrim",
+			"Elder Scrolls Online",
+			"Blades",
+			"Call to Arms",
+			"Dawnstar",
+			"Legends",
+			"Oblivion Mobile",
+			"Shadowkey",
+			"Skyrim Pinball",
+			"Stormhold"
+		],
+		"genres": [
+		    "Role Playing"
```

Comment:
As with `companies`, `genres` shouldn't be included.

Reply:
Note: As someone unfamiliar with the project it would be good to include all this information in the Readme.

Reply:
Thanks, I'll update it!